.github/workflows/tag_image_push_rhel.yml

```
@@ -142,6 +141,14 @@ jobs:
           release_version: ${{ env.RELEASE_VERSION }}
           is_lts_override: ${{ inputs.IS_LTS_OVERRIDE }}
 
+      - name: Delete unpublished images
+        if: inputs.DRY_RUN != 'true'
+        run: |
+          VERSION=${RELEASE_VERSION}-jdk${{ matrix.jdk }}
+          source .github/scripts/publish-rhel.sh
+
+          delete_unpublished_images "$RHEL_PROJECT_ID" "$VERSION" "$RHEL_API_KEY"
```

```suggestion
          delete_unpublished_images "${RHEL_PROJECT_ID}" "${VERSION}" "${RHEL_API_KEY}"
```

Nit - the preferred style by shellcheck. I know we aren't consistent with this...